src/Lykke.Cqrs.MessageCancellation/Controllers/MessageCancellationController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Lykke.Cqrs.MessageCancellation.Models;
+using Lykke.Cqrs.MessageCancellation.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Lykke.Cqrs.MessageCancellation.Controllers
+{
+    [PublicAPI, Route("/api/plugin/message-cancellation")]
+    public class MessageCancellationController : Controller
```

Comment:
This controller must be registered. Need to use something like here - https://github.com/LykkeCity/Lykke.Sdk/blob/master/src/Lykke.Sdk/LykkeServiceCollectionContainerBuilderExtensions.cs#L79